src/components/Footer/index.js

```diff
@@ -14,14 +14,14 @@ const COMPLIANCE = [
 ]
  const COMPLIANCE2 =[
 
-  {img:'bestchange', url: "https://bestchange.com" , name: 'bestchange'},
-  {img: 'okchanger', url: "https://okchanger.com" , name: 'okchanger'},
+  {img:'bestchange', url: "https://www.bestchange.net/nexchange-exchanger.html" , name: 'bestchange'},
```

Comment:
why bestchange.net? also should be RU for RU customer @chiragneb

Reply:
@mrtrader1313

Reply:
Ok fixing